runtime/oti/j9consts.h

```diff
@@ -328,9 +328,9 @@ extern "C" {
 #define J9_RUNTIME_EXTENDED_METHOD_BLOCK 0x200000
 #define J9_RUNTIME_SHUTDOWN_STARTED 0x400000
 #define J9_RUNTIME_EXIT_STARTED 0x800000
-#define J9_RUNTIME_AGGRESSIVE_VERIFICATION 0x1000000
+#define J9_RUNTIME_UNUSED_2 0x1000000
 #define J9_RUNTIME_NO_PRIORITIES 0x2000000
-#define J9_RUNTIME_DFPBD 0x4000000
+#define J9_RUNTIME_SHOW_HIDDEN_FRAMES 0x4000000
```

Comment:
I see you've removed / repurposed existing bits.  Have you validated that these bits are used by existing code?  Sadly there may be code that's depending on the bit value and not the named constant....

Reply:
inspected all codes that accesses `vm->runtimeFlags`, didn't see any hardcoded values used with the flag. Sanity testing passed, extended testing passed before rebasing, retesting now

Reply:
extended tests passed